### PR TITLE
Review markup and fix images

### PR DESCRIPTION
**Pull request by outoftardis**
This PR mostly contains the following changes:

- Fixed markup/punctuation
- Added steps for procedures
- Added todo comments
- For images:
  - Renamed image files to give them more meaningful names (not for all images)
  - Removed duplicates
  - Centered and minimized images in docs (not for all images)
  - Checked that the image was relevant to the text it was accompanying (100% for renamed images, can't say the same for all the other).

I did review the doc contents and in some places changed the phrasing a bit, but it was a secondary focus.

### CODE REVIEW

File: src/advanced-trading.md [hunk 1]
```diff
@@ -28,61 +26,77 @@ We recommend using the SORA testnet for practice exercises. Here are the Testnet
...
-* *Network fee* - SORA fee for processing the transaction
+* *VAL PER XOR*: The price of 1 XOR in VAL.
+* *XOR PER VAL*: The price of 1 VAL in XOR.
+* *Minimum Received*: The minimum number of destination assets that you'll get after the swap.
+* *Liquidity provider fee*: The fee that liquidity provider will take. In this case, it's an XYK Pool.
+* *Network fee*: SORA fee for processing the transaction (gas).
 
 #### What Insufficient _liquidity_ means
 
 You can get the error saying "**Not enough liquidity**". It's important to understand what this means.
```
Comment (WRRicht3r) [inline src/advanced-trading.md]:
```suggestion
It is possible to get an error saying "**Not enough liquidity**" and understanding what this means is important.
```

File: src/check-supply.md [hunk 2]
```diff
@@ -4,82 +4,83 @@
...
+    :::
 
-![](</.gitbook/assets/Screenshot2021-04-29at13.56.40.png>)
+    ![](</.gitbook/assets/check-supply-fill-in-asset-id.png>)
 
-The results are shown for XOR:
+    The results are shown for XOR:
 
-353.0005 kXOR = 353 000.5 XOR (including XOR on Ethereum network)
+    353.0005 kXOR = 353 000.5 XOR (including XOR on Ethereum network)
```
Comment (WRRicht3r) [inline src/check-supply.md:44]:
```suggestion
    353.0005 kXOR = 353 000.5 XOR (including XOR on the Ethereum network)
```

File: src/consensus.md [hunk 3]
```diff
@@ -83,13 +86,13 @@ Intel(R) Core(TM) i7-7700K CPU @ 4.20GHz
 Uptime of 99.9%
 ```
 
-**If you’re interested in helping secure the future of SORA and Polkaswap, and earning rewards in the process, read more** [**here**](https://medium.com/sora-xor/how-to-run-a-sora-testnet-node-a4d42a9de1af)
+If you’re interested in helping secure the future of SORA and Polkaswap, and earning rewards in the process, read more [here](https://medium.com/sora-xor/how-to-run-a-sora-testnet-node-a4d42a9de1af).
```
Comment (WRRicht3r) [inline src/consensus.md]: Can we link this to the wiki article we have on running a node?

**Review by WRRicht3r — CHANGES_REQUESTED** [review]

File: src/consensus.md (hunk 3 above)
Comment (outoftardis) [reply src/consensus.md]: sure, I'll change the link

**Review by WRRicht3r — APPROVED** [review]

**Review by bgodlin — APPROVED** [review]

**Pull request merged**